business_requirement/models/business.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import except_orm
+
+
+class BusinessRequirement(models.Model):
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _name = "business.requirement"
+    _description = "Business Requirement"
+    _order = 'id desc'
+
+    @api.model
+    def _get_default_company(self):
+        company_id = self.env.user._get_company()
+        if not company_id:
+            raise except_orm(
+                _('Error!'),
+                _('There is no default company for the current user!'))
+        return self.env['res.company'].browse(company_id)
+
+    sequence = fields.Char(
+        'Sequence',
+        readonly=True,
+        copy=False,
+        index=True,
+    )
+    name = fields.Char(
+        'Name',
+        required=False,
+        readonly=True,
+        copy=False,
+        states={'draft': [('readonly', False)]}
+    )
+    description = fields.Char(
+        'Description', required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    business_requirement = fields.Html(
+        'Customer Story',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    scenario = fields.Html(
+        'Scenario',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    gap = fields.Html(
+        'Gap',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    category_ids = fields.Many2many(
+        'business.requirement.category',
+        string='Categories',
+        relation='business_requirement_category_rel',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    state = fields.Selection(
+        selection="_get_states",
```

Comment:
@elicoidal  @victormartinelicocorp  we can keep it like this since the module is still in WIP. The migration is still needed even if we change the name now. :)